### ISSUE TITLE
The query in slow log can't be parsed when there single comment in sql

### ISSUE THREAD

**Issue by lilinghai** [pingcap/tidb#30886]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

If we execute the sql with tidb, and image it is a slow query.
```sql
select count(*) from t x ,t y
-- condition
where x.a<y.a
and x.a is not null -- comment
```
The query recorded in tidb slow log is ```select count(*) from t x ,t y -- condition where x.a<y.a  and x.a is not null -- comment;``` , and we can't parse the sql correctly because we can't identify which sentences are commented.
Compared with tidb，the mysql slow query in the log is
```
select count(*) from t x ,t y
-- condition
where x.a<y.a
and x.a is not null -- comment;
```
### 2. What did you expect to see? (Required)
same with mysql
### 4. What is your TiDB version? (Required)

master

**Comment by Tangruilin** [comment]
@wjhuang2016 I'd like to try to fix this, are there any one do this?

**Comment by djshow832** [comment]
> @wjhuang2016 I'd like to try to fix this, are there any one do this?

No. If you see it's not assigned, you can assign it to yourself. @Tangruilin
